eng/pipelines/templates/steps/build-test.yml

```diff
@@ -7,12 +7,20 @@ parameters:
   OSName: ''
   BeforeTestSteps: []
   BuildTargetingString: 'azure-*'
-
+  RunMyPy: and(ne(variables['PythonVersion'], '2.7'), ne(variables['PythonVersion'], 'pypy3'))
```

Comment:
There is no prescribed action here, only conflicting opinions loosely held...

The age old DRY vs. tight coupling fight... The argument for keeping as is would be that (almost) all uses of `build-test.yml` need this particular expression and it's nice to only have to go one place to change it. One argument against keeping this as is would be that it couples this implementation to a global variable which is defined outside of the parameters of this template (one exception to this rule is for things like `$(Build.WhateverDirectory)` which can be assumed to always be present in this context for DevOps).

Reply:
Yeah. I'll circle back to this after making certain that this guy works properly with:

* `livetests`
* can run the world in an hour